From 568c5d686ff3cb21782a28d26592c8a96856bf04 Mon Sep 17 00:00:00 2001
From: rafaelrc7 <contact@rafaelrc.com>
Date: Tue, 5 Aug 2025 19:35:17 -0300
Subject: [PATCH] [PATCH] Fix sdrpp breaking every time the package is rebuilt

---
 core/src/server.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/src/server.cpp b/core/src/server.cpp
index 501198c3..439516c1 100644
--- a/core/src/server.cpp
+++ b/core/src/server.cpp
@@ -78,6 +78,10 @@ namespace server {
         // Load config
         core::configManager.acquire();
         std::string modulesDir = core::configManager.conf["modulesDirectory"];
+        {
+            std::size_t pos = modulesDir.find("@prefix@");
+            if (pos != std::string::npos) modulesDir.replace(pos, 8, INSTALL_PREFIX);
+        }
         std::vector<std::string> modules = core::configManager.conf["modules"];
         auto modList = core::configManager.conf["moduleInstances"].items();
         std::string sourceName = core::configManager.conf["source"];
--
2.50.1

